src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -1413,5 +1422,78 @@ private static ExternalProjectReference ToExternalProjectReference(PackageSpec p
                 msbuildProjectPath: null,
                 projectReferences: Enumerable.Empty<string>());
         }
+
+        private static class TraceEvents
+        {
+            private const string EventNameBuildAssetsFile = "RestoreCommand/BuildAssetsFile";
+            private const string EventNameBuildRestoreGraph = "RestoreCommand/BuildRestoreGraph";
+            private const string EventNameCalcDependencySpec = "RestoreCommand/CalcDependencySpec";
```

Comment:
A more accurate name would be `CalcNoOpRestore`.

Reply:
Fixed.